widgetastic_manageiq/__init__.py

```diff
@@ -4878,3 +4878,55 @@ class MigrationDashboardStatusCard(AggregateStatusCard):
         ' h2[contains(@class,"card-pf-title")  and contains(.,{@name|quote})]]'
     )
     TITLE_ANCHOR = ".//h2"
+
+
+class AllAlertsList(Widget):
+    """Represents the list of fired alerts on the Monitor->All Alerts page"""
+
+    ROOT = ParametrizedLocator(".//div[contains(@class,{@list_class|quote})]")
+    ITEM_LOCATOR = './/div[contains(@class,"list-group-item alert")]'
+    ITEM_DESCRIPTION_LOCATOR = './/span[contains(@class,"description-column")]'
+    ITEM_EXPAND_LOCATOR = './/span[contains(@class,"fa-angle-right")]'
```

Comment:
If you add `fa-angle-right` to widgetastic_patternfly.utils.IconConstants` you can use `PFicon.icons.ANGLE_RIGHT` here.